.circleci/config.yml

```
@@ -3,35 +3,42 @@
 version: 2.1
 
 commands:
+  ##################
+  # CircleCI Commands
+  ##################
   circleci_configure_path:
-    description: Fix for CI prefixing the PATH if you set it via env vars. DO NOT DO THAT. See commit description or details.
+    description: Fix for CI prefi xing the PATH if you set it via env vars. DO NOT DO THAT. See commit description or details.
```

Looks like you made an accidental edit here.

```suggestion
    description: Fix for CI prefixing the PATH if you set it via env vars. DO NOT DO THAT. See commit description or details.
```